HeliosEmbedded/Makefile

```diff
@@ -96,7 +96,6 @@ DEFINES=\
 
 CFLAGS = -g \
 	 -Os \
-	 -MMD \

```

Comment:
Why did you remove this, this generates the .d files which are used to detect changes to the code and properly perform partial rebuilds